config/passport.js

```diff
@@ -493,6 +493,80 @@ passport.use(
   ),
 );
 
+/**
+ * Sign in with Microsoft using OAuth2Strategy.
+ */
+const microsoftStrategyConfig = new OAuth2Strategy(
+  {
+    authorizationURL: 'https://login.microsoftonline.com/common/oauth2/v2.0/authorize',
+    tokenURL: 'https://login.microsoftonline.com/common/oauth2/v2.0/token',
+    clientID: process.env.MICROSOFT_CLIENT_ID,
+    clientSecret: process.env.MICROSOFT_CLIENT_SECRET,
+    callbackURL: `${process.env.BASE_URL}/auth/microsoft/callback`,
+    scope: ['openid', 'profile', 'email', 'User.Read'].join(' '),
```

Comment:
Add a comment that to get a refresh token we can add  'offline_access' to the scope list.
Trade-off: They are going to see permission approval every time that they try to login with the offline_access as one part of the scope